tests/unit/test_cli.py

```diff
@@ -53,14 +64,17 @@ async def test_entrypoint_dry_run():
         "cou.cli.generate_plan"
     ) as mock_generate_plan, patch("cou.cli.apply_plan"), patch("cou.cli.Analysis.create"), patch(
         "builtins.print"
-    ) as mock_print:
+    ) as mock_print, patch(
+        "cou.cli.apply_plan"
+    ) as mock_apply_plan:
```

Comment:
not-blocking: Can we move all this mocking as decorators? It will be much more readable like this.

Reply:
I've changed to use the `mocker` module.